setup.cfg

```diff
@@ -86,7 +86,7 @@ zip_ok = false
 [flake8]
 extend-select = B950
 # TODO: don't disable D*, fix up issues instead
-ignore = N801,N802,N803,E203,E226,E305,W504,E252,E301,E302,E501,E704,W503,W504,F811,D1,D4
```

Comment:
What's this rule? How is it related to changing a test name?

Reply:
This rule catches cases where there are multiple definitions of the same thing - cases like this are quite common in tests where someone copy/pastes a function to add a new variant, but then forgets to rename the test - which results in only one test being run.

If you run pyflakes/ruff with the F811 check enabled on master you'll see:

```
./tests/test_urldispatch.py:1154:1: redefinition of unused 'test_invalid_route_name' from line 1149
```

I've removed the ignore here so this doesn't happen again, and this is the only instance of it currently in Aiohttp

Reply:
We also pay a lot more attention to codecov these days, which should also stop this happening again.

Reply:
@alexmac makes sense. We should be moving towards having less ignored rules and more enabled ones. I was mostly asking because there's no code comment explaining it. These days I preach only disabling linting rules accompanied by an explanation so it doesn't linger around for years with nobody knowing why.